app/src/main/java/com/mediaan/mymediaan/viewModel/MainNavigationViewModel.kt

```diff
@@ -114,6 +114,7 @@ class MainNavigationViewModel(application: Application) : AndroidViewModel(appli
             avatarIcon = avatarIcon
         )
         sharedPrefsUtil.saveLoggedInUserId(getApplication<Application>().applicationContext, profile.id)
+        saveLoggedInUserProfile(profile)
```

Comment:
Are you sure this call is needed here? 

It is now duplicated. We call `createNewProfile` in CreateProfileScreen and then within same coroutine call `viewModel.saveLoggedInUserProfile(newProfile)`. Maybe it's correct to leave only one call?

Reply:
Nice catch, forgot to remove this after debugging. Was trying to see if the order of execution would matter in the coroutine. (it did not, lol)